patches/chromium/fix_desktop_capturer_show_own_window.patch

```diff
@@ -0,0 +1,26 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: George Xu <george.xu@slack-corp.com>
+Date: Wed, 11 Dec 2024 15:00:09 -0800
+Subject: fix: desktopCapturer.getSources does not return electron window on
+ Windows
+
+This patch is to fix issue https://github.com/electron/electron/issues/36037, where `desktopCapturer.getSources` does not return the electron window on Windows.
```

Comment:
Could we add a CL link to the original deadlock issue here, just to ensure that future folks can debug why this patch was needed if it ends up being an issue?